protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -232,6 +233,34 @@ message ListCommitBlobsRequest {
     }
 }
 
+// Getting experimentRuns linked to a commit
+message ListCommitExperimentRunsRequest {
+    RepositoryIdentification repository_id = 1;
+    Pagination pagination = 2;
+    string commit_sha = 3;
+
+    message Response {
+        repeated ExperimentRuns runs = 1;
+        int64 total_records = 2;
+    }
+}
+
+// Getting experimentRuns linked to a blob
+message ListBlobExperimentRunsRequest {
+    RepositoryIdentification repository_id = 1;
+    Pagination pagination = 2;
+    string commit_sha = 3;
+    string blob_sha = 4;
+    // List of folder names (with the last element potentially representing the blob name)
+    // This will be used to limit the search to scope just this location, like `git log /my/path` would do
+    repeated string location_prefix = 5;
```

Comment:
The user doesn't know the blob sha (it's internal information right now) and we don't want to support full location prefixes. The location should map to a single blob.
```suggestion
    repeated string location = 4;
```